src/topNavigation/_topNavigation.scss

```diff
@@ -1,3 +1,29 @@
 .TopNavigation {
   z-index: 500;
 }
+
+.TopNavigation_FavouritesIcon {
+  position: relative;
+  color: black;
+  text-decoration: none;
+}
+
+.TopNavigation__FavouritesIcon__FavCount {
+  position: absolute;
+  left: 15px;
+  top: -10px;
+  background-color: $color-gold;
+  border-radius: 50%;
+  width: 23px;
```

Comment:
I'd try to find suitable sizing variables to replace these bare numbers here if possible, but if not, then these should be fine.